test/Bounty.test.js

```diff
@@ -66,7 +66,7 @@ contract('Bounty', function ([_, owner, researcher, nonTarget]) {
       await this.bounty.claim(this.targetAddress, { from: researcher });
       const claim = await this.bounty.claimed();
 
-      claim.should.eq(true);
+      claim.should.be.true;
```

Comment:
does this not throw an "unused result" linting error in your editor? Mine always prefers that there's a function call rather than just a dereference that implicitly triggers a function call.

Anyway, instead of .be.true I very much prefer `.equal(true)` for clarity.

Reply:
Nope, I don't have that kind of setup :/

I went for `should.be.true` for consistency, but have no strong feelings to be honest, either is fine by me. @frangio?

Reply:
Note that we disable that specific feature here: https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/test/.eslintrc

Reply:
lmao we totally do, I think I added it to avoid having to change the tests within the eslint commit